### PR TITLE
Add warning about committing the Carthage directory.

### PR DESCRIPTION
**Pull request by bbewan**
https://app.asana.com/0/353548986073982/461852772803841

### CODE REVIEW

**Comment by bbdocs** [comment]
See the website here: http://99-bb-docs.surge.sh

File: builds/dependencies/carthage.adoc [hunk 1]
```diff
@@ -7,6 +7,16 @@ In order to guarantee repeatability in builds -- we recommend you check
 in both your `Cartfile` and `Cartfile.resolved` files into your source
 control.
 
+[IMPORTANT]
+===========
+When Carthage downloads dependencies, they appear in the `Carthage`
+directory. If you commit this directory to your repository, **buddybuild
+does not run the `carthage` command** because it assumes that you have
+taken responsibility for managing the Carthage dependencies to optimize
+your build time.
```
Comment (coryalder) [inline builds/dependencies/carthage.adoc]: Hmm - I'm not sure we want to say this last bit.

"to optimize your build time." implies that they *should* be doing this for faster builds and I'm not sure that an idea we want to encourage. Plus there are lots of other reasons for them to want to manage their own carthage deps.

Is it alright if we go with just "taken responsibility for managing the Carthage dependencies."?

Comment (bbewan) [reply builds/dependencies/carthage.adoc]: Good suggestion. Thanks!

**Comment by bbdocs** [comment]
See the website here: http://99-bb-docs.surge.sh

**Pull request merged**